src/JuMP.jl

```diff
@@ -495,6 +498,66 @@ end
 getValue(arr::Array{QuadExpr}) = map(getValue, arr)
 
 ##########################################################################
+# SOC
+# Container for √(∑ expr)
+type GenericNorm{C,V}
```

Comment:
I was thinking of making it parameterized and having the typealias be ``Norm2``.